conf/messages.en

```diff
@@ -232,3 +232,13 @@ businessTradingNameWithoutID.error.required = Enter the trading name of your bus
 businessTradingNameWithoutID.error.invalid = Trading name must only include letters a to z, numbers 0 to 9, ampersands (&), apostrophes, backslashes, carets (^), grave accents (`), hyphens and spaces
 businessTradingNameWithoutID.error.length = Trading name must be 80 characters or less
 businessTradingNameWithoutID.change.hidden = Change your business trading name
+
+whatIsYourNINumber.title = What is your National Insurance number?
+whatIsYourNINumber.heading = What is your National Insurance number?
+whatIsYourNINumber.checkYourAnswersLabel = Your National Insurance number
+whatIsYourNINumber.checkYourAnswersLabel.hiddentText = your National Insurance number
```

Comment:
This key should be `whatIsYourNINumber.change.hidden` and the text should be `Change your National Insurance number`